docker/web/config/Caddyfile

```diff
@@ -0,0 +1,50 @@
+{
+	cache
+}
+
+(s3path) {
+	route {args[0]} {
+		cache
+		rewrite {args[0]} {args[1]}
+		s3proxy {
+			bucket {$S3_BUCKET}
+			region auto
```

Comment:
Is it possible to use spaces for indention? Seems like `.editorconfig` should enforce that

https://github.com/philomena-dev/philomena/blob/2a0a885f34fbed5a6f7138721430752df85f89af/.editorconfig#L1-L9